<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>q_settings</class>
 <widget class="QWidget" name="q_settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="m_lb_title">
     <property name="text">
      <string>Android id: NONE | Status: disconnected</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="m_hl_to_apply">
     <item>
      <widget class="QGroupBox" name="m_gb_settings">
       <property name="title">
        <string>Settings</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="m_lb_samples">
            <property name="text">
             <string>Samples</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_cb_samples">
            <item>
             <property name="text">
              <string>8000</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>16000</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>24000</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="m_gb_output">
       <property name="title">
        <string>Output file</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="m_lb_name">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_pb_rename">
            <property name="text">
             <string>rename</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="m_pb_apply">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_gb_player">
     <property name="title">
      <string>Player</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCustomPlot" name="m_plotter" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="m_hl_player">
        <item>
         <widget class="QGroupBox" name="m_gb_play_stop">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QCheckBox" name="m_cb_play_stop">
             <property name="text">
              <string>PLAY</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="m_gb_volume">
          <property name="title">
           <string>Volume</string>
          </property>
          <layout class="QHBoxLayout" name="m_hl_volume">
           <item>
            <widget class="QSlider" name="m_hs_sound">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="m_pb_back">
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_pb_back</sender>
   <signal>clicked()</signal>
   <receiver>q_settings</receiver>
   <slot>back_to_device_list()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_cb_play_stop</sender>
   <signal>clicked()</signal>
   <receiver>q_settings</receiver>
   <slot>play_or_pause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_hs_sound</sender>
   <signal>actionTriggered(int)</signal>
   <receiver>q_settings</receiver>
   <slot>volume(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pb_rename</sender>
   <signal>clicked()</signal>
   <receiver>q_settings</receiver>
   <slot>rename()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pb_apply</sender>
   <signal>clicked()</signal>
   <receiver>q_settings</receiver>
   <slot>apply_settings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>back_to_device_list()</slot>
  <slot>apply_settings()</slot>
  <slot>rename()</slot>
  <slot>play_or_pause()</slot>
  <slot>stop()</slot>
  <slot>volume(int)</slot>
 </slots>
</ui>
